packetDecoder.go

```diff
@@ -62,23 +63,18 @@ func handlePacket(packet gopacket.Packet) {
 			log.Println("IPv4: ", decodedPacket.ipv4.SrcIP, "->", decodedPacket.ipv4.DstIP)
 
 		case layers.LayerTypeTCP:
-			decodedPacket.FlowHash = hashCombine(decodedPacket.NetFlow.FastHash(), decodedPacket.TCP.TransportFlow().FastHash())
+			flowHash := computeFlowHash(decodedPacket.NetFlow)
+			decodedPacket.FlowHash = flowHash
 		}
 	}
 }
 
-// based on boost::hash_combine
-// http://www.boost.org/doc/libs/1_63_0/boost/functional/hash/hash.hpp
-func hashCombine(h, k uint64) uint64 {
-	m := uint64(0xc6a4a7935bd1e995)
-	r := uint64(47)
-
-	k *= m
-	k ^= k >> r
-	k *= m
-
-	h ^= k
-	h *= m
-
-	return h + 0xe6546b64
+// Since FastHash cannot be used as a key, this function computes hash of Flow object
+// which can be used as a key.
+// Ref: https://godoc.org/github.com/google/gopacket#Flow
+func computeFlowHash(flow gopacket.Flow) []byte {
+	h := sha256.New()
+	byteTemp := fmt.Sprintf("%v", flow)
```

Comment:
use Log instead of fmt to output to stdout, will make file-based logging support easier in future